specification/dimensions/billingperiodstart.md

```diff
@@ -0,0 +1,32 @@
+# Billing Period Start
+
+Billing period represents the time window for which you have or will receive an invoice for. Billing period represents the time window for which you have or will receive an invoice for. The time window is inclusive of the start date and exclusive of the end date.
+
+Billing Period Start represents the beginning of a billing period.
+
+The BillingPeriodStart column MUST be present in the billing data. This column MUST conform to [FOCUS Date/Time Requirements](#date/timerequirements) and MUST NOT contain null values. The sum of the Billed Cost metric for line items in a given billing period MUST match the total cost of the invoices received for that billing period.
+
+## Column ID
+
+BillingPeriodStart
```

Comment:
Is anyone else have a nagging thought every time you see this that it should be "BillingPeriodStart**Date**"? For some reason, I can't look at this without feeling like it's a typo.

Reply:
I did think about this, but my take is that BillingPeriodStart works fine. If we say Date, it might mean it is a column of type Date and not Datetime. Less confusing this way.

Reply:
Can. Shorter name would be nice. It feels like Date would imply its not a DateTime per the expectation?